plugins/inputs/stackdriver/stackdriver.go

```diff
@@ -608,12 +607,24 @@ func (s *Stackdriver) gatherTimeSeries(
 			tags[k] = v
 		}
 
+		slashIdx := strings.LastIndex(tsConf.measurement, "/")
```

Comment:
This takes the last segment of the metric_name URL from stackdriver and makes it the `metric_category`, it's here to combat duplicate trailing segments in stackdriver metric names.

e.g.:

`cloudsql.googleapis.com/database/memory/quota`

and

`cloudsql.googleapis.com/database/disk/quota`

With this patch, the metric into circonus will still be called `quota` but will be tagged with `metric_category:disk` or `metric_category:memory` in this example.